framework/test/integration/node/processor/block_process/process_block.spec.ts

```diff
@@ -0,0 +1,404 @@
+/*
```

Comment:
It would've been better if the file name was not changed or the change is detected by the github as it's hard to see the change

Reply:
the history is preserved now